src/invoker.js

```diff
@@ -7,14 +7,14 @@ var curryN = require('./curryN');
  * Turns a named method with a specified arity into a function
  * that can be called directly supplied with arguments and a target object.
  *
- * The returned function is curried and accepts `len + 1` parameters where
+ * The returned function is curried and accepts `arity + 1` parameters where
  * the final parameter is the target object.
  *
  * @func
  * @memberOf R
  * @category Function
  * @sig (Number, String) -> (a... -> c -> b)
```

Comment:
Yes, that's probably the best we can do, although even '`Object`' might be replaced with '`*`'.  This is as weakly typed as I think we get in Ramda.  Give us a number and a string, and we'll give you back a function from that many parameters plus some specific something, returning something else.

... Sure, dude.

However, it's also quite useful.